pom.xml

```diff
@@ -76,6 +76,12 @@
             <version>0.0.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I noticed that it was in test scope after I sent you the note.

That is fine. I only care about delivery dependencies.  Test dependencies
are always fine if they add value.



On Tue, Jun 2, 2015 at 9:26 AM, rayortigas <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/tdunning/t-digest/pull/56#discussion_r31541575>:
>
> > @@ -76,6 +76,12 @@
> >              <version>0.0.5</version>
> >              <scope>test</scope>
> >          </dependency>
> > +        <dependency>
>
> Sorry, I only brought it into test scope for convenience methods to verify
> serialization and deserialization. I can remove the dep and inline the
> serialization/deserialization code in the test.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tdunning/t-digest/pull/56/files#r31541575>.
>